src/main/java/org/folio/linked/data/integration/kafka/KafkaMessageListener.java

```diff
@@ -35,16 +36,27 @@ public class KafkaMessageListener {
     concurrency = "#{folioKafkaProperties.listener['data-import-instance-create'].concurrency}",
     topicPattern = "#{folioKafkaProperties.listener['data-import-instance-create'].topicPattern}")
   public void handleDataImportInstanceCreatedEvent(List<ConsumerRecord<String, DataImportEvent>> consumerRecords) {
-    consumerRecords.forEach(consumerRecord -> {
-      log.info("Received: {}", consumerRecord);
-      var event = consumerRecord.value();
-      if (requiredHeadersProvided(consumerRecord.headers())) {
-        tenantScopedExecutionService.executeAsyncTenantScoped(consumerRecord.headers(),
-          () -> dataImportEventHandler.handle(event));
-      } else {
-        log.warn("Received DataImportEvent with no required Folio headers will be ignored: {}", event);
-      }
-    });
+    consumerRecords.forEach(this::processRecord);
+  }
+
+  private void processRecord(ConsumerRecord<String, DataImportEvent> consumerRecord) {
+    log.info("Received event: {}", consumerRecord);
+    var event = consumerRecord.value();
+    if (!requiredHeadersProvided(consumerRecord.headers())) {
```

Comment:
minor
suggest not to use `!`
instead, we can remove the method `isNotContainsRequerdHeaders"

Reply:
Done